### ISSUE TITLE
game.data.version is deprecated in V9 in favor of game.version

### ISSUE THREAD

**Issue by schlosrat** [ChadKeating/rolladvantage-token-stamp-2-foundry#14]
FYI you've got a reference to game.data.version in rolladvantage-token-stamp-2-foundry.js.

This is causing a warning message in the console when your module is included and the game world is launched or relaunched with Foundry V9 build 242 (probably any V9 build)

You should be able to safely replace 'game.data.version' with 'game.version' to get the behavior you want without the warning message, though this may cause an issue for folks not using recent builds of FVTT.

**Comment by RollAdvantage** [comment]
Yeah there was a minor change to make it compatible with v9 so I didnt want to release two version and want to maintain compatiblity as much as possible for now. The `game.data.version` check is only relevant for <v9 versions, but I guess I need to add a null check if it's being removed.

Thanks for the heads up though, I only briefly tested on v9 as I'm using v8 still while my campaign finishes.
